preferencesfx/src/main/java/com/dlsc/preferencesfx/PreferencesFx.java

```diff
@@ -241,11 +238,38 @@ public PreferencesFx addEventHandler(EventType<PreferencesFxEvent> eventType, Ev
    *
    * @param eventType    the event type from which to unregister
    * @param eventHandler the handler to unregister
-   *
    * @throws NullPointerException if either event type or handler are {@code null}.
    */
   public PreferencesFx removeEventHandler(EventType<PreferencesFxEvent> eventType, EventHandler<? super PreferencesFxEvent> eventHandler) {
     preferencesFxModel.removeEventHandler(eventType, eventHandler);
     return this;
   }
+
+  /**
+   * Returns a PreferencesFxView, so that it can be used as a Node.
+   *
+   * @return a PreferencesFxView, so that it can be used as a Node.
+   */
+  public PreferencesFxView getView() {
+    return preferencesFxView;
+  }
+
+  /**
+   * Saves all of the changed settings.
+   * <p>
+   * Call this method when the preferences are shown by using {@link #getView()}.
+   */
+  public void saveSettings() {
+    preferencesFxModel.saveSettings();
+  }
+
+  /**
+   * Discard all of the changed settings.
+   * <p>
+   * Call this method when the preferences are shown by using {@link #getView()}
```

Comment:
My proposal here would be: `Call this method to undo all changes made in the settings when showing the preferences by using {@link #getView()}.`

Reply:
Thanks, for the proposition. I changed it :)